src/SFA.DAS.ApplyService.Database/Tables/AssessorPageReviewOutcome.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE TABLE [dbo].[AssessorPageReviewOutcome](
+	[Id] [uniqueidentifier] NOT NULL,
+	[ApplicationId] [uniqueidentifier] NOT NULL,
+	[SequenceNumber] [int] NOT NULL,
+	[SectionNumber] [int] NOT NULL,
+	[PageId] [nvarchar](50) NOT NULL,
+	[Assessor1UserId] NVARCHAR(256) NULL,     
+	[Assessor1ReviewStatus] NVARCHAR(20) NULL, 
+    [Assessor1ReviewComment] NVARCHAR(MAX) NULL,
+	[Assessor2UserId] NVARCHAR(256) NULL, 
+	[Assessor2ReviewStatus] NVARCHAR(20) NULL,
+	[Assessor2ReviewComment] NVARCHAR(MAX) NULL,
+	[CreatedAt] DATETIME2 NOT NULL, 
```

Comment:
Add default for CreatedAt

ALTER TABLE [dbo].AssessorPageReviewOutcome ADD  DEFAULT (getdate()) FOR [CreatedAt]

Reply:
For all other tables we are adding CreatedAt (and UpdatedAt) in the ApplyRepository.
Do you still want me to add it the DB?
And for all other cases GETUTCDATE() is used.
Which one I should use GETUTCDATE() or GETDATE() ?

Reply:
ALTER TABLE [dbo].AssessorPageReviewOutcome ADD DEFAULT (getutcdate()) FOR [CreatedAt] added